nknd.go

```diff
@@ -156,16 +156,30 @@ func nknMain(c *cli.Context) error {
 
 	var wallet vault.Wallet
 	var account *vault.Account
-	for wallet == nil {
-		time.Sleep(time.Second * 3)
+	if config.Parameters.WebGuiCreateWallet {
+		for wallet == nil {
```

Comment:
I think it's better to use `wallet == nil || account == nil` just in case get wallet success but get account fails